resources/definitions/hypothesis-opener.yaml

```diff
@@ -0,0 +1,49 @@
+assets:
+    css:
+        - buttons.css
+        - hypothesis-opener.css
+    js: []
+schema:
+    $schema: 'http://json-schema.org/draft-04/schema#'
+    type: object
+    properties:
+        button:
+            $schema: 'http://json-schema.org/draft-04/schema#'
+            type: object
+            allOf:
+                -
```

Comment:
don't know the conventions here, but is the newline after `-` needed? e.g. https://github.com/elifesciences/api-raml/blob/8972b5fcfc67331a332319e459b4073249660c97/src/model/community-list.v1.yaml#L28-L34

Reply:
This file is an auto-generated, expanded version the equivalent file brought over from file in `pattern-library`. In `pattern-library`, this is the current convention. Happy to change it to harmonise across repos, but suggest this is out of scope for this PR.

Reply:
Yes, follow the established convention (which I don't know around here)